notebooks/Location Sharing Status.ipynb

```diff
@@ -0,0 +1,197 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    gis = GIS("home", verfiy_cert=False)
</pre><p><code>verfiy_cert</code> should be <code>verify_cert</code>.</p><p> </p><p>Should we be using <code>True</code> instead, or omit altogether given default is <code>True</code>? </p><p>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/field-maps-scripts/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
removed the argument and added a comment